tests/foreman/smoke/test_smoke.py

```diff
@@ -67,17 +70,24 @@ def test_installer_check_services(self):
             'pulp_workers',
             'qdrouterd',
             'qpidd',
-            'tomcat',
+            'tomcat6' if major_version == RHEL_6_MAJOR_VERSION else 'tomcat',
         )
 
         # check `services` status using service command
-        if get_host_info()[1] >= 7:
+        if major_version >= RHEL_7_MAJOR_VERSION:
             status_format = 'systemctl status {0}'
         else:
             status_format = 'service {0} status'
 
         for service in services:
             result = ssh.command(status_format.format(service))
+            if (major_version == RHEL_6_MAJOR_VERSION and
+                    service is 'qpidd' and
+                    not bz_bug_is_open(1246152)):
+                # This is a note to fix this test once Bug 1246152 is fixed
+                self.fail('Bug 1246152 is fixed. Fix Me.')
+            else:
```

Comment:
It is necessary because I dont want lines 91 and 92 to execute if service is 'qpidd'